i18n/en/docusaurus-plugin-content-docs/current/about/understanding/naming.md

```diff
@@ -4,59 +4,60 @@ sidebar_position: 4
-- If you ask for help in the community, you will get an answer to your question faster, **if you use the same terminology**
+Naming conflicts can occur when terms used in the **Feature-Sliced Design** methodology overlap with terms used in the business:
 
-## When can naming interfere?
+- `FSD#process` vs simulated process in an application,
+- `FSD#page` vs log page,
+- `FSD#model` vs car model.
 
-When developing a project for displaying/building/modeling any processes, or developing an application for the layout of magazine pages, you may face the problem that **the terms used in the methodology overlap with the terms that your business operates**.
+For example, a developer seeing the word process in the code will spend unnecessary time trying to understand which process they are talking about.
 
-- `FSD#process` vs simulated process in your application
-- `FSD#page` vs magazine page
-- `FSD#model` vs car model
+When communicating within the development team, when saying the word process, everyone involved should clearly understand what we're talking about, the process as a business entity or the process from Feature-Sliced Design.
 
-<!-- TODO: think about examples for other terms -->
+When communicating with the business, developers sometimes use technical terms that the business isn't familiar with. So a developer using the term process to refer to a process from Feature-Sliced Design will introduce a misunderstanding into the conversation which may require additional time to clarify.
 
-Such name collisions can negatively affect the development process.
+<!-- TODO: think of examples for other terms -->
 
-- The developer, seeing the word `process` in the code, will spend extra time understanding which process is being discussed
+These conflicts can disrupt the development process because developers may spend extra time trying to figure out exactly what the conversation is about.
 
-- When communicating within the development team, saying the word `process`, all participants in the conversation should clearly understand what is being discussed, about the process as a business entity or about the process from **Feature-Sliced Design**.
-  
-- When communicating with business, developers sometimes use technical terms that the business is not familiar with. So the developer, using the term `process`, referring to the process from **Feature-Sliced Design**, will introduce a misunderstanding into the conversation, which may require additional time for clarification
+In addition, when communicating with the business, it is important to be aware of any technical terms that may be unfamiliar to non-technical stakeholders. Using such terms can lead to misunderstandings that may take additional time to clarify.
 
-## See also
+:::note
+It is important that all team members clearly understand the meaning of terms such as "process, model, ..." when communicating within the team.
+:::
```

Comment:
suggestion: I'd also suggest removing this note, I think it's not so much a note as a summary of the preceding text

Reply:
I agree with you